### PR TITLE
TC6-ContactUs

### PR DESCRIPTION
**Pull request by marwafayez103**

### CODE REVIEW

File: src/test/resources/file.docx
Comment (MahmoudElSharkawyGS) [inline src/test/resources/file.docx:1]: Please add a descriptive name this this file, and it's better to be in directory with a descriptive name also.

File: src/main/java/com/gizasystems/automationexercise/pages/NavigationBar.java [hunk 1]
```diff
@@ -6,13 +6,15 @@
 
 public class NavigationBar {
     // Variables
-    private SHAFT.GUI.WebDriver driver;
+    private final SHAFT.GUI.WebDriver driver;
```
Comment (MahmoudElSharkawyGS) [inline src/main/java/com/gizasystems/automationexercise/pages/NavigationBar.java:9]: Why added final here?

File: src/test/resources/testDataFiles/ContactUs.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "Name": "Marwa",
+  "Email": "mm@gmail.com",
```
Comment (MahmoudElSharkawyGS) [inline src/test/resources/testDataFiles/ContactUs.json:3]: We agreed not to add our names in the test data :D

**Review by MahmoudElSharkawyGS — CHANGES_REQUESTED** [review]
- Please make sure to add the pipeline execution link to the PR
- Please make sure to format your code

File: src/test/resources/file.docx
Comment (marwafayez103) [reply src/test/resources/file.docx:1]: done

File: src/main/java/com/gizasystems/automationexercise/pages/ContactUsPage.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    private final By email = By.name("email");
+    private final By subject = By.name("subject");
+
+    private final By Message=By.id("message");
+
+    private final By submitBtn = By.name("submit");
+
+    private final By uploadBtn = By.xpath("//input[@type='file']");
+
+    private final By success_msg= By.xpath("//div[@class='status alert alert-success']");
```
[the remark this answers is not in the document]
Comment (marwafayez103) [reply src/main/java/com/gizasystems/automationexercise/pages/ContactUsPage.java]: done

File: src/main/java/com/gizasystems/automationexercise/pages/NavigationBar.java (hunk 1 above)
Comment (marwafayez103) [reply src/main/java/com/gizasystems/automationexercise/pages/NavigationBar.java]: done

File: src/test/resources/testDataFiles/ContactUs.json (hunk 2 above)
Comment (marwafayez103) [reply src/test/resources/testDataFiles/ContactUs.json]: done

**Comment by marwafayez103** [comment]
@MahmoudElSharkawyGS the pieline run https://github.com/MahmoudElSharkawyGS/AutomationExercisePractice/actions/runs/7828641256 
but it failed in firfox only i tried alot also with mostafa and i cant find solution :D

**Review by MahmoudElSharkawyGS — APPROVED** [review]

**Pull request merged**